src/ucm/util/reloc.c

```diff
@@ -312,12 +312,52 @@ static ucs_status_t ucm_reloc_apply_patch(ucm_reloc_patch_t *patch,
     return ctx.status;
 }
 
+/* read serinfo from 'module_path', result buffer must be destroyed
+ * by free() call */
+static Dl_serinfo *ucm_dlopen_load_serinfo(const char *module_path)
+{
+    Dl_serinfo *serinfo = NULL;
+    Dl_serinfo stub;
+    void *caller_module;
```

Comment:
caller_module -> module
this function does not know it's caller so remove all "caller" mentions in this function

Reply:
ok, renamed

Reply:
renamed